account_journal_report/tests/test_account_journal_report.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests.common import TransactionCase
+
+
+class TestAccountJournalReport(TransactionCase):
+    """
+        Tests for Account Journal Report.
+    """
+    def setUp(self):
+        super(TestAccountJournalReport, self).setUp()
+
+        self.res_users_model = self.env['res.users']
+        self.company = self.env.ref('base.main_company')
+        self.g_account_user = self.env.ref('account.group_account_user')
+
+        self.user = self._create_user('user_1', [self.g_account_user],
+                                      self.company).id
+
+        self.report_model = \
+            self.env['report.account_journal_report.journal_ledger']
+        self.wiz = self.env['account.journal.entries.report']
+        self.report_name = 'account_journal_report.journal_ledger'
+        self.report_xlsx_name = 'account_journal_report.journal_ledger_xlsx'
+        self.report_title = 'Journal Ledger'
+
+    def _create_user(self, login, groups, company):
+        group_ids = [group.id for group in groups]
+        user = self.res_users_model.create({
+            'name': login,
+            'login': login,
+            'password': 'demo',
+            'email': 'example@yourcompany.com',
+            'company_id': company.id,
+            'company_ids': [(4, company.id)],
+            'groups_id': [(6, 0, group_ids)]
+        })
+        return user
+
+    def test_account_journal_report(self):
+
+        wiz_id = self.wiz.create({})
```

Comment:
Well, it's a safeguard, because you have to make your tests state-consistent, and if you depend on the default values of the wizard, maybe any change can make this change (for example, a custom module that changes these defaults in a customer).